README.md

```diff
@@ -130,6 +130,13 @@ Help:
 5. Check if everything worked on the test.pypi.org site.
 6. Publish it for real: `twine upload dist/*`
 
+### Releasingi automatically
+
+You can release a new version by workflows and automatically.
+
+1. Bump up the version number in `paci/__init__.py` according to [semver](https://semver.org/).
+2. Create a new release tag version and it will start our pipeline and push the new version to PyPI and PyPI test.
```

Comment:
```suggestion
2. Create a new release tag version and it will start our pipeline and push the new version to PyPI and PyPI test after it is merged to main.
```